packages/api-v2/src/APIProvider.tsx

```diff
@@ -31,6 +31,7 @@ type APIContextData = {
     subscribe: TSubscribeFunction;
     unsubscribe: TUnsubscribeFunction;
     queryClient: QueryClient;
+    setOnReconnected: (onReconnected: () => void) => void;
```

Comment:
Does this type work better than `React.Dispatch<React.SetStateAction<VoidFunction>>`? It sure looks more simple

Reply:
hey, noted this and will have a look in next PR (coming soon), I do not want to loose reapprovals ;)